ctags/main/routines.h

```diff
@@ -16,16 +16,96 @@
 
 #include <stdio.h>
 
-#include "mio.h"
+#include "vstring.h"
 
 /*
 *   MACROS
 */
+#define xMalloc(n,Type)    (Type *)eMalloc((size_t)(n) * sizeof (Type))
+#define xCalloc(n,Type)    (Type *)eCalloc((size_t)(n), sizeof (Type))
+#define xRealloc(p,n,Type) (Type *)eRealloc((p), (n) * sizeof (Type))
+
 #define ARRAY_SIZE(X)      (sizeof (X) / sizeof (X[0]))
 
 #define STRINGIFY(X) STRINGIFY_(X)
 #define STRINGIFY_(X) #X
 
+/*
+ *  Portability macros
+ */
+#ifndef PATH_SEPARATOR
+# if defined (MSDOS_STYLE_PATH)
+#  define PATH_SEPARATOR '\\'
+# else
+#  define PATH_SEPARATOR '/'
+# endif
+#endif
+
+#if defined (MSDOS_STYLE_PATH) && defined (UNIX_PATH_SEPARATOR)
+# define OUTPUT_PATH_SEPARATOR	'/'
+#else
+# define OUTPUT_PATH_SEPARATOR	PATH_SEPARATOR
+#endif
+
+/*
+*   DATA DECLARATIONS
+*/
+#if defined (MSDOS_STYLE_PATH)
+extern const char *const PathDelimiters;
```

Comment:
ah, it's fixed in 7e0845b083e56f59b87390507df9444c4f99b826 (on which my comment somehow shows as outdated, but I don't get why)